### DISCUSSION TITLE
How to modify parameters according to different subcatchment?

### DISCUSSION THREAD

**Discussion by BastianT22** [Deltares/Wflow.jl#296] [Q&A] [answered]
I can modify parameters by "cfmax.calue = x.x". It will work in the whole basin. I have divided basin into 11 sub-basins and numbered them. I would like to know how to modify the parameters for the 11 sub-basins to different values in the toml file.

**Comment by JoostBuitink** [comment, marked as answer]
This is not directly possible in Wflow. The best way to do this, is to manually modify your staticmaps file as a preprocessing step, and add a new layer with different `cfmax` values for each subbasin. This way, you create a new 2D map in the staticmaps. which you can then refer to in the toml file (similar to the other spatially distributed parameters). 
We typically use python and xarray for these type of operations, but any programming language and/or library should be able to this.

**Reply by BastianT22** [reply]
That's all right. Thanks for your reply!
